use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,358 @@
+1. [Verify Intel QAT works in a Docker\* Kata container](#verify-qat)
+    1. [Build OpenSSL Intel QAT engine container](#verify-qat-1)
+    1. [Test Intel QAT in Docker](#verify-qat-2)
+    1. [Troubleshooting](#verify-qat-3)
+1. [Optional Scripts](#optional-scripts)
+    1. [Verify Intel QAT card counters are incremented](#optional-scripts-1)
+    1. [Unbind all vf’s from vfio driver, reset, and rebind to the host](#optional-scripts-2)
+
+##Script variables<a name="script-variables"></a>
+The variables listed below must be set before running through the scripts. These variables refer to locations to store modules and configuration files on the host and as well as links to the drivers to use. Modify these as needed to point to updated drivers or different install locations.
+
+###Set environment variables (Every Reboot)<a name="script-variables-1"></a>
+```
+export QAT_CONF_LOCATION=~/QAT_conf
+export QAT_DRIVER_URL=https://01.org/sites/default/files/downloads//qat1.7.l.4.5.0-00034.tar.gz
+export QAT_DOCKERFILE=https://raw.githubusercontent.com/intel/intel-device-plugins-for-kubernetes/master/demo/openssl-qat-engine/Dockerfile
+export QAT_SRC=~/src/QAT
+export GOPATH=~/src/go
+export OSBUILDER=~/src/osbuilder
+export KATA_KERNEL_LOCATION=~/kata
+export KATA_ROOTFS_LOCATION=~/kata
+```
+
+##Prepare the Host (Clear Linux)<a name="prepare-host"></a>
+The host could be a bare metal instance or a virtual machine. If using a virtual machine, make sure that KVM nesting is enabled.  All of the instructions below reference an Intel QAT.  Some of the instructions must be modified if using a different Intel QAT device. You can identify the Intel QAT chipset by executing the following.
+
+###Identify which PCI Bus the Intel QAT card is on<a name="prepare-1"></a>
+```
+for i in 0434 0435 37c8 1f18 1f19; do lspci -d 8086:$i; done
+b1:00.0 Co-processor: Intel Corporation C62x Chipset QuickAssist Technology (rev 04)
+	b3:00.0 Co-processor: Intel Corporation C62x Chipset QuickAssist Technology (rev 04)
+	b5:00.0 Co-processor: Intel Corporation C62x Chipset QuickAssist Technology (rev 04)
+```
+
+###Install necessary bundles for Clear Linux and disable kernel updates<a name="prepare-2"></a>
+The Clear Linux installer generally installs the os-core, os-core-update, and openssh-server bundle. Add the following minimum bundles including the Linux kernel headers to complete the rest of this guide. The Linux kernel headers are needed to compile the out of tree kernel modules to support Intel QAT on the host. The os-clr-on-clr is a large development bundle that can be installed instead to include everything below with additional items. This guide installs the 4.19 LTS kernel from 2018 because the crypto modules are set to "m" and not "y" which allows them to be replaced with the out of tree driver. Otherwise you will need to build your own Clear Linux kernel with the QAT drivers compiled in.
```

Comment:
All non-dictionary words like os-clr-on-clr and 4.19 need to be put in backticks to avoid incurring the wrath of the [spellchecker](https://github.com/kata-containers/tests/tree/master/cmd/check-spelling): `os-clr-on-clr`, `4.19`, `m`, `y`.

- [dictionary](https://github.com/kata-containers/tests/tree/master/cmd/check-spelling/data): What do we think about adding "crypto" to our dictionary folks? It's a common enough contraction of "cryptography" but maybe we should always just always spell it in full ?

Reply:
I've raised a PR for some other terms so comment in either place folks: https://github.com/kata-containers/tests/pull/1764.